### PR TITLE
Updates to HTML/CSS for headings and other elements

### LINKED ISSUE #93: HTML / CSS review
We have some drift away from SearchWorks, and a couple of heading levels to adjust.

* [x] I truly mean these to be global styles across SW, Requests, MyAccount, and whatever else lives in this family:
```
body:
     font-family: 'Source Sans Pro', 'Arial Unicode MS', Helvetica, sans-serif;
     font-size: 14px;
     background-color: #fbfbf9;
     line-height:1.428571429;
     color: black;

h1, h2
     font-weight: 300;

h1
     color: #3f3c30;
     font-size: 2.3em;
     letter-spacing: -0.05em;

h2
     color: #8c1515;
     font-size: 1.6em;
     letter-spacing: 0;
     line-height: 1.2em;

a, a:hover, a:focus 
     color: black;
     text-decoration: none;
     border-bottom: 1px dotted #b6b1a9;
```

* [x] fix the vertical spacing of the top banner: #topnav-container AND .topnav-divider should have 'margin-top: 0`

* [x] h2 on home page:
```
     border-bottom: 1px solid #e3dfd5;
     margin: .5em 0;
```

* [x] h2 in form
```
     margin: 0 0 1em 0;
```

* [x] buttons throughout: add `font-size 1.1em`
* [x] 2-line buttons: second line is `font-size .9em` (so 2-line buttons need a `<span>` instead of a `<br>`)

* [x] the h3 around the name in the form should be an h2
**Nice sr-only text, Jessie :-)**

* [x] remove the parens from the email address when the form doesn't display a name. 
(If this is harder than it should be, then we can remove the parens in both contexts.)

Going to make a separate issue for the request deliver/scan page.

### PR DESCRIPTION
**Pull request by jchristo4**
Closes #93

### CODE REVIEW

File: app/assets/stylesheets/base.scss [hunk 1]
```diff
@@ -11,26 +11,36 @@ a {
...
 }
 
 h2 {
   border-bottom: 1px solid $sul-h2-border-color;
-  color: $cardinal-red;
+  color: $sul-h2-font-color;
+  font-size: 1.6em;
   letter-spacing: 0;
   line-height: 1.2em;
   margin: 0.5em 0;
```
Comment (jkeck) [inline app/assets/stylesheets/base.scss]: @jvine is the intention to have the `<h2>` styled globally to have the underline  (i.e. the SearchWorks title and email below)?

It seems that #93 describes that the underline should only actually apply to the home page.

h2s

Comment (jvine) [inline app/assets/stylesheets/base.scss]: Correct, the underline should only be on the home page. 

It also looks like the name above the button is using the global h2 margin, when it should be  margin: 0 0 1em 0; - 4th bullet in #93.

Comment (jchristo4) [inline app/assets/stylesheets/base.scss]: @jkeck, @jvine 

* Removed `h2` underline from non-home pages
* Added margin to form `h2`

image

Comment (jvine) [inline app/assets/stylesheets/base.scss]: :+1:

**Pull request merged**